server/sonar-web/src/main/js/apps/permission-templates/views/UsersView.js

```diff
@@ -17,14 +17,37 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import _ from 'underscore';
 import Modal from '../../../components/common/modals';
 import Template from '../templates/permission-templates-users.hbs';
 import '../../../components/SelectList';
+import {
+    addProjectCreatorToTemplate,
+    removeProjectCreatorFromTemplate
+} from '../../../api/permissions';
 
 export default Modal.extend({
   template: Template,
 
+  events () {
+    return {
+      ...Modal.prototype.events.apply(this, arguments),
+      'change #grant-to-project-creators': 'onCheckboxChange'
+    };
+  },
+
+  onCheckboxChange() {
+    const checked = this.$('#grant-to-project-creators').is(':checked');
+    if (checked) {
+      addProjectCreatorToTemplate(
+          this.options.permissionTemplate.name,
+          this.options.permission.key);
+    } else {
+      removeProjectCreatorFromTemplate(
+          this.options.permissionTemplate.name,
+          this.options.permission.key);
+    }
+  },
+
   onRender () {
     Modal.prototype.onRender.apply(this, arguments);
     const searchUrl = window.baseUrl + '/api/permissions/template_users?ps=100&permission=' +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3512)